fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerAppointments.js

```diff
@@ -149,7 +156,7 @@ function AdminAccounts() {
             }
         });
 
-        fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments${params.toString() && "?" + params.toString()}`, {
+        fetch(`http://localhost:8080/api/v1/appointments${params.toString() && "?" + params.toString()}`, {
```

Comment:
You forgot to chanhe the url to ${process.env.REACT_APP_BASE_URL}

Reply:
Oups thanks for reminding me i will change it.